docs/rescal-snow-tutorial.md

```diff
@@ -2,6 +2,38 @@
 
 ![](example_images/snowfall_example.gif)
 
+snowfall-wind phase diagram
+
+The runs with the higher snowfall rate have a much deeper average snow depth than the runs with lower snowfall rate. The runs with high wind speed (low Tau_min) have less even snow cover, with better-defined ripples and dunes.
+
+    Parallel run errors?
+
+        msub/sbatch commands not allowed by your computing cluster -> seek support from someone familiar with the cluster
+        permissions errors -> run chmod u+rwx \* in test_parallel_runs; contact administrator if this is disallowed
+        script could not find input (run.run or run.par or sealevel_snow.prop) -> rerun param_space_exploration_example.py and confirm that it produced the output above; check that you're running msub/sbatch from scripts; check relative directory references in msub/sbatch script and submit.sh
+        runs time out before producing useful output -> increase walltime; large simulations on slow machines may ultimately take a few hours
```

Comment:
This is poorly formatted and hard to read/follow

Reply:
From the line numbers, these were part of the copy-paste error.

Reply:
The text belongs at the bottom of the tutorial (where it now is, as part of the parallel parameter space exploration example). There, it's formatted more helpfully with bullet points:

>*Parallel run errors?*
>
> - msub/sbatch commands not allowed by your computing cluster -> seek support from someone familiar with the cluster
> - permissions errors -> run `chmod u+rwx \*` in test_parallel_runs; contact administrator if this is disallowed
> - script could not find input (run.run or run.par or sealevel_snow.prop) -> rerun param_space_exploration_example.py and confirm that it produced the output above; check that you're running msub/sbatch from scripts; check relative directory references in msub/sbatch script and submit.sh
> - runs time out before producing useful output -> increase walltime; large simulations on slow machines may ultimately take a few hours